cardinal/config.go

```diff
@@ -0,0 +1,28 @@
+package cardinal
+
+import "os"
+
+type WorldConfig struct {
+	RedisAddress       string
+	RedisPassword      string
+	CardinalWorldId    string
+	CardinalPort       string
+	CardinalDeployMode string
+}
+
+func GetWorldConfig() WorldConfig {
+	return WorldConfig{
+		RedisAddress:       getEnv("REDIS_ADDRESS", "localhost:6379"),
+		RedisPassword:      getEnv("REDIS_PASSWORD", ""),
+		CardinalWorldId:    getEnv("CARDINAL_WORLD_ID", "world"),
+		CardinalPort:       getEnv("CARDINAL_PORT", "3333"),
```

Comment:
the default cardinal port is 4040. are we doing 3333 or 4040? if we want to go with 3333, we should update the docs and comments in this PR. additionally, in cardinal/server/server.go in the Initialize method, we load this env variable again. that should be removed if we're doing this here.

Reply:
Didn't realize we decided on 4040. Will update this and World CLI accordingly